h2o-py/demos/uplift_drf_demo.ipynb

```diff
@@ -0,0 +1,2844 @@
+    "\n",
+    "$\\sqrt{X}(P, Q) = \\sum_{i=0}^{N} \\frac{\\sqrt{p_i-q_i}}{q_i}$\n",
+    "\n",
+    "where:\n",
+    "\n",
+    "- $P$ is treatment group distribution\n",
+    "\n",
+    "- $Q$ is control group distribution\n",
+    "\n",
+    "In a tree node the result value for a split is sum: $metric(P, Q) + metric(1-P, 1-Q)$. \n",
+    "\n",
+    "For the split gain value, the result within the node is normalized using a gini coefficient (Eclidean or ChiSquared) or entropy (KL) for each distribution before and after the split.\n",
+    "\n",
+    "\n",
+    "**Uplift score in each leaf is calculated as:**\n",
+    "\n",
+    "- $TP = (TY1 + 1) / (T + 2)$\n",
+    "- $CP = (CY1 + 1) / (C + 2)$\n",
+    "- $uplift\\_score = TP - CP $\n",
+    "\n",
+    "where:\n",
+    "- $T$ how many observations in a leaf are from the treatment group (how many data rows in a leaf have ``treatment_column`` label == 1) \n",
+    "- $C$ how many observations in a leaf are from the control group (how many data rows in the leaf have ``treatment_column`` label == 0)\n",
+    "- $TY1$ how many observations in a leaf are from the treatment group and respond to the offer (how many data rows in the leaf have ``treatment_column`` label == 1 and ``response_column`` label == 1)\n",
+    "- $CY1$ how many observations in a leaf are from the control group and respond to the offer (how many data rows in the leaf have ``treatment_column`` label == 0 and ``response_column`` label == 1)\n",
+    "\n",
+    "**Note**: A higher uplift score means more observations from treatment group respond to the offer than from control group. Which means offered treatment has positive effect. The uplift score can be negative, if more observations from control group respond to the offer without treatment.\n",
+    "\n",
+    "<br>\n",
+    "<br>\n",
+    "\n",
+    "![Difference between SDT and Uplift DT](https://blog.h2o.ai/wp-content/uploads/2022/01/tree.png)\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## H2O Implementation (Major release 3.36)\n",
+    "\n",
+    "The H2O-3 implementation of Uplift DRF is based on DRF because the principle of training is similar to DRF. It is tree based uplift algorithm. Uplift DRF generates a forest of classification uplift trees, rather than a single classification tree. Each of these trees is a weak learner built on a subset of rows and columns. More trees will reduce the variance. Classification takes the average prediction over all of their trees to make a final prediction. \n",
+    "\n",
+    "Currently, in H2O-3 only binomial trees are supported, as well as the uplift curve metric and Area Under Uplift curve (AUUC) metric and the Qini coefficient. We are working on adding also regression trees and more metrics, for example, normalized AUUC, and more. "
```

Comment:
Are you sure with Qini coefficient? I remember you refactor it to Qini metric? What is the difference actually?